fl4health/clients/basic_client.py

```diff
@@ -832,7 +832,8 @@ def update_after_step(self, step: int) -> None:
         """
         Hook method called after local train step on client. step is an integer that represents
         the local training step that was most recently completed. For example, used by the APFL
-        method to update the alpha value after a training a step.
+        method to update the alpha value after a training a step. Also used by the Moon, Fenda
```

Comment:
Since MOON, FENDA, MK-MMD are acronyms, we can capitalize them in this comment. Similarly Ditto can be capitalized as a proper name.